modules/logging-es-deploy-cli.adoc

```diff
@@ -0,0 +1,319 @@
+----
+<1> You must specify the `openshift-operators-redhat` namespace. The `openshift-operators`
+namespace might contain Community Operators, which are untrusted and could publish
+a metric with the same name as an {product-title} metric, which would cause
+conflicts.
+<2> A string value that specifies the label as shown to ensure that cluster monitoring scrapes the `openshift-operators-redhat` namespace.
+
+. Apply the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `Namespace` object for the {clo}:
++
+.Example `Namespace` object
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging # <1>
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-monitoring: "true"
+----
+<1> You must specify `openshift-logging` as the namespace for logging versions 5.7 and earlier. For logging 5.8 and later, you can use any namespace.
+
+. Apply the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create an `OperatorGroup` object for the {es-op}:
++
+.Example `OperatorGroup` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-operators-redhat
+  namespace: openshift-operators-redhat # <1>
+spec: {}
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+
+. Apply the `OperatorGroup` object by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Create a `Subscription` object to subscribe a namespace to the {es-op}:
++
+--
+include::snippets/logging-stable-updates-snip.adoc[]
+--
++
+.Example `Subscription` object
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: elasticsearch-operator
+  namespace: openshift-operators-redhat # <1>
+spec:
+  channel: <channel> # <2>
+  installPlanApproval: Automatic # <3>
+  source: redhat-operators # <4>
+  sourceNamespace: openshift-marketplace
+  name: elasticsearch-operator
+----
+<1> You must specify the `openshift-operators-redhat` namespace.
+<2> Specify `stable`, or `stable-<x.y>` as the channel.
+<3> `Automatic` allows the Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available. `Manual` requires a user with appropriate credentials to approve the Operator update.
+<4> Specify `redhat-operators`. If your {product-title} cluster is installed on a restricted network, also known as a disconnected cluster, specify the name of the `CatalogSource` object you created when you configured the Operator Lifecycle Manager (OLM)
+
+. Apply the subscription by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <filename>.yaml
+----
+
+. Verify the Operator installation by running the following command:
++
+[source,terminal]
+----
+$ oc get csv --all-namespaces
```

Comment:
the good command to be only listed the elasticsearch csv should be:

`oc get csv --all-namespaces|grep elasticsearch`